apps/spruce/src/pages/adminSettings/tabs/GeneralTab/OtherTab/schemaFields.ts

```diff
@@ -0,0 +1,548 @@
+      },
+    },
+  },
+  uiSchema: {
+    "ui:ObjectFieldTemplate": CardFieldTemplate,
+    "ui:data-cy": "jira-notifications",
+    "ui:objectFieldCss": objectGridCss,
+    customFields: {
+      "ui:addButtonText": "Add New Jira Project",
+      "ui:data-cy": "jira-custom-fields-list",
+      "ui:orderable": false,
+      "ui:fullWidth": true,
+      "ui:fieldCss": fullWidthCss,
+      "ui:arrayItemCSS": arrayItemCSS,
+      items: {
+        fields: {
+          "ui:addButtonText": "Add Field",
+          "ui:data-cy": "jira-fields-list",
+          "ui:orderable": false,
+          "ui:fullWidth": true,
+          "ui:fieldCss": fullWidthCss,
+          "ui:arrayItemCSS": arrayItemCSS,
+        },
+        components: {
+          "ui:widget": widgets.ChipInputWidget,
+        },
+        labels: {
+          "ui:widget": widgets.ChipInputWidget,
+        },
+      },
+    },
+  },
+};
+
+export const spawnHost = {
+  schema: {
+    unexpirableHostsPerUser: {
+      type: "number" as const,
+      title: "Unexpirable Hosts Per User",
+    },
+    unexpirableVolumesPerUser: {
+      type: "number" as const,
+      title: "Unexpirable Volumes Per User",
+    },
+    spawnHostsPerUser: {
+      type: "number" as const,
+      title: "Spawn Hosts Per User",
+    },
+  },
+  uiSchema: {
+    "ui:ObjectFieldTemplate": CardFieldTemplate,
+    "ui:data-cy": "spawn-host",
+    "ui:objectFieldCss": objectGridCss,
+  },
+};
+
+export const sleepSchedule = {
+  schema: {
+    permanentlyExemptHosts: {
+      type: "array" as const,
+      title: "Permanently Exempt Hosts",
+      items: {
+        type: "string" as const,
+      },
+    },
+  },
+  uiSchema: {
+    "ui:data-cy": "sleep-schedule",
+    "ui:ObjectFieldTemplate": CardFieldTemplate,
+    "ui:objectFieldCss": objectGridCss,
+    permanentlyExemptHosts: {
+      "ui:widget": widgets.ChipInputWidget,
+      "ui:fieldCss": fullWidthCss,
+    },
+  },
+};
+
+export const tracerConfiguration = {
+  schema: {
+    collectorEndpoint: {
+      type: "string" as const,
+      title: "Collector Endpoint",
+      format: "validURL",
+    },
+    collectorInternalEndpoint: {
+      type: "string" as const,
+      title: "Collector Internal Endpoint",
+      format: "validURL",
+    },
+    collectorAPIKey: {
+      type: "string" as const,
+      title: "Collector API Key",
+    },
+    enabled: {
+      type: "boolean" as const,
+      title: "Enabled",
+    },
+  },
+  uiSchema: {
+    "ui:ObjectFieldTemplate": CardFieldTemplate,
+    "ui:data-cy": "tracer-configuration",
+    "ui:objectFieldCss": objectGridCss,
+  },
+};
+
+export const projectCreationSettings = {
+  schema: {
+    totalProjectLimit: {
+      type: "number" as const,
+      title: "Total Project Limit",
+    },
+    repoProjectLimit: {
+      type: "number" as const,
+      title: "Repository Project Limit",
+    },
+    jiraProject: {
+      type: "string" as const,
+      title: "Jira Project",
+    },
+    repoExceptions: {
```

Comment:
oop one more thing, I think there is a duplicate of this as a section

<img width="1001" height="225" alt="Screenshot 2025-08-19 at 5 52 46 PM" src="https://github.com/user-attachments/assets/a7fecfb1-5ffc-42cb-ac36-62af1c403975" />